### PR TITLE
Chore/try runtime ci

### PR DESCRIPTION
**Pull request by simonsso**

### CODE REVIEW

**Comment by simonsso** [comment]
@aliXsed @Hounsette  I will revisit the last two actions-rs addons later.

This is ready to be reviewed now.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,18 +14,41 @@ env:
   try-runtime-uri: wss://eden-rpc.dwellir.com:443
 
 jobs:
-  test:
+  clippy_lint:
+    runs-on: ubuntu-latest-8-cores
+    steps:
+      - uses: actions/checkout@v3
+      - uses: simonsso/setup-rust-toolchain@3b513b9b4098bc5ff24c930d279f9501375a1a09
```
Comment (aliXsed) [inline .github/workflows/ci.yml:21]: What is now different in your fork of this action?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -14,18 +14,41 @@ env:
...
+        with:
+          toolchain: ${{ env.nightly }}
+          target: ${{ env.target }}
+          components: clippy
```
Comment (aliXsed) [inline .github/workflows/ci.yml:25]: Do we still need the highlighted lines any more?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (simonsso) [reply .github/workflows/ci.yml:21]: It is a mistake in my merge.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (simonsso) [reply .github/workflows/ci.yml:25]: It should not be the fork but the source of the fork.

**Review by aliXsed — APPROVED** [review]

**Pull request merged**